dtfs-central-api/api-tests/v1/utilisation-reports/get-fee-record.api-test.ts

```diff
@@ -0,0 +1,132 @@
+import { Response } from 'supertest';
+import { HttpStatusCode, getUri } from 'axios';
+import { Bank, FeeRecordEntityMockBuilder, RECONCILIATION_IN_PROGRESS, ReportPeriod, UtilisationReportEntityMockBuilder } from '@ukef/dtfs2-common';
+import { withSqlIdPathParameterValidationTests } from '@ukef/dtfs2-common/test-cases-backend';
+import { testApi } from '../../test-api';
+import { SqlDbHelper } from '../../sql-db-helper';
+import { GetFeeRecordResponseBody } from '../../../src/v1/controllers/utilisation-report-service/get-fee-record.controller';
+import { mongoDbClient } from '../../../src/drivers/db-client';
+import { wipe } from '../../wipeDB';
+import { aBank } from '../../../test-helpers';
+
+interface CustomResponse extends Response {
+  body: GetFeeRecordResponseBody;
+}
+
+console.error = jest.fn();
+
+const BASE_URL = '/v1/utilisation-reports/:reportId/fee-records/:feeRecordId';
+
+describe(`GET ${BASE_URL}`, () => {
```

Comment:
I'm struggling to find or understand which controller/function this is testing. Can you please confirm?

Reply:
This is testing `dtfs-central-api/src/v1/controllers/utilisation-report-service/get-fee-record.controller/index.ts` [here](https://github.com/UK-Export-Finance/dtfs2/pull/3916/files#diff-9cbfc4fa41c2992b6f97a90c6f4dc1735e040176af3952e0cfb24a9fc56da1f8)